--- liveusb-creator-3.11.6/liveusb/gui.py.old	2011-11-02 21:00:31.000000000 +0400
+++ liveusb-creator-3.11.6/liveusb/gui.py	2011-12-25 15:02:50.172616810 +0400
@@ -102,14 +102,14 @@
     """
     def start(self, filename=None, url=None, basename=None, size=None,
               now=None, text=None):
-        self.emit(QtCore.SIGNAL("maxprogress(int)"), size)
+        self.emit(QtCore.SIGNAL("maxprogress(int)"), size/2)
 
     def update(self, amount_read, now=None):
         """ Update our download progressbar.
 
         :read: the number of bytes read so far
         """
-        self.emit(QtCore.SIGNAL("progress(int)"), amount_read)
+        self.emit(QtCore.SIGNAL("progress(int)"), amount_read/2)
 
     def end(self, amount_read):
         self.update(amount_read)
